devices/parser/management/commands/sensor.py

```diff
@@ -43,7 +43,7 @@ def parse_payload(self, payload):
         for meas in payload:
             postfix = meas["sensor"][-1:]
 
-            time = parse_time(meas["observations"][0]["timestamp"]).astimezone(pytz.UTC)
+            time = parse_time(meas["observations"][0]["timestamp"], dayfirst=False).astimezone(pytz.UTC)
```

Comment:
I assume `dayfirst` should be True here. Here is a fresh time string from a few seconds ago (2nd March, 2021):

```python
>>> parse_time("02/03/2021T10:43:16UTC").astimezone(pytz.UTC)
datetime.datetime(2021, 2, 3, 10, 43, 16, tzinfo=<UTC>)
>>> parse_time("02/03/2021T10:43:16UTC", dayfirst=False).astimezone(pytz.UTC)
datetime.datetime(2021, 2, 3, 10, 43, 16, tzinfo=<UTC>)
>>> parse_time("02/03/2021T10:43:16UTC", dayfirst=True).astimezone(pytz.UTC)
datetime.datetime(2021, 3, 2, 10, 43, 16, tzinfo=<UTC>)  # <-- this is correct value
```

Reply:
Fixing.